src/test/scala/org/broadinstitute/hail/utils/OrderedRDDSuite.scala

```diff
@@ -151,6 +151,22 @@ class OrderedRDDSuite extends SparkSuite {
         .collect()
         .forall { case (v1, v2) => v1 == v2 }
     }
+
+    property("partitionReordering") = Prop.forAll(sorted) { case (nPar, is) =>
+
+      val rdd = sc.parallelize(is, nPar)
+      val shuffledIndices = Array.range(0, nPar).sortBy(_ => scala.util.Random.nextDouble())
```

Comment:
This is a annoying change since the Prop/Gen syntax is so bad, but it would be natural to have `Gen.shuffle(Gen.const(0 until nPar))` as another generator item.